docs/objects.inv

```diff
@@ -1,5 +1,5 @@
 # Sphinx inventory version 2
 # Project: seeq-plot-curve
-# Version: 0.1.0
+# Version: 0.1.1
 # The remainder of this file is compressed using zlib.
 xڅ�AN!E���[�.FM&1�rBC�&���)g�5��'�� �wԫ��=)o���̍	Z�RWx5ь�B;R�߄+�R	�y�QT#������QXGS�L8���$�;Gwimf�v�^u�AR��m8b�A���?���B���/���e#Y�%��/�ƫֈ@��G荂8������׎�68��#����h�3My�A-zڊ��y��^Y)�r�ə�Ce�9�r���f���
```

Comment:
I am not familiar with how documentation is build, but what is this seemingly binary line here?